eviatar/enable_windows_features.ps1

```diff
@@ -0,0 +1,33 @@
+﻿function Install-WindowsFeatures {
+    param (
+        [Parameter(Mandatory = $true)]
+        [string[]]$Features
+    )
+    
+    foreach ($Feature in $Features) {
+        Write-Host "Installing feature: $Feature"
+        Install-WindowsFeature -Name $Feature -IncludeAllSubFeature -IncludeManagementTools -Verbose
+    }
+}
+
+function Rename-Server {
+    param (
+        [Parameter(Mandatory = $true)]
+        [string]$NewName
+    )
+
+    Write-Host "Renaming server to: $NewName"
+    Rename-Computer -NewName $NewName -Force
+}
+
+$features = Read-Host "Enter the list of Windows Features to install (comma-separated)"
+$newName = Read-Host "Enter the new server name"
+
+$featuresArray = $features -split ","
+
+Install-WindowsFeatures -Features $featuresArray
+
+Rename-Server -NewName $newName
+
+Write-Host "Restarting the server..."
+Restart-Computer -Force

```

Comment:
The Rename-Computer CmdLet has a parameter that aloows you to rename the host and perform restart in one command

Reply:
True, Rename-Computer has a -Restart parameter that I can use to restart the computer after renaming, which simplifies the code by removing the need for a separate restart command.